db/schema.rb

```diff
@@ -68,6 +80,27 @@
     t.integer  "phase_id"
   end
 
+  create_table "file_types", force: :cascade do |t|
```

Comment:
looks like a bunch of defunct tables have leaked back into the schema.rb file.

you may be able to just manually remove `file_types` `file_uploads` and `splash_logs`

Reply:
Removed those tables from schema.rb so it's now the same as under development branch but with the extra tables needed for conditionals and it looks like it still works fine.